GruntTasks/Targets/concat.orchestra_js.js

```diff
@@ -121,6 +122,9 @@ module.exports = {
     'web/built/openorchestrabackoffice/js/user/UserFormView.js',
     'web/built/openorchestrauseradmin/js/user/*.js',
 
+    //--[ CONTENT TYPE ]--//
+    'web/built/openorchestrabackoffice/js/contentType/TableViewAction.js',
```

Comment:
I think it would be better to add this to the TableView section, but as you want